### PR TITLE
(SIMP-8630) Documentation updates

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Fixed:
  - Updated/corrected documentation
  - Added REFERENCE.md
  - Removed the unused libreswan::use_certs parameter
  - Converted libreswan::params to module data
  - Hooked service notifications at the class levels

SIMP-8630 #close

### CODE REVIEW

**Review by op-ct — APPROVED** [review]

**Comment by op-ct** [comment]
Content looks good; [`pkg:compare_latest_tag` is angry][0] that the *.pp files have been changed since 3.4.0

[0]: https://travis-ci.com/github/simp/pupmod-simp-libreswan/jobs/426974891#L492-L510

**Review by op-ct — APPROVED** [review]
:eye: 👍

File: README.md [hunk 1]
```diff
@@ -33,27 +39,30 @@ Please read our [Contribution Guide](https://simp.readthedocs.io/en/stable/contr
...
+
+* All `ipsec.conf` options can be found in `ipsec.conf(5)`.
 
 
 ## Setup
+
 * Ensure the libreswan and NSS packages are available.
 
-Before installing pupmod-simp-libreswan make sure you read the libreswan documentation thouroughly.  It is located at https://libreswan.org/wiki/Introduction. After reading the introduction select the Main Wiki Page link to get to the user documentation:
+Before installing pupmod-simp-libreswan make sure you read the libreswan documentation thoroughly.  It is located at https://libreswan.org/wiki/Introduction. After reading the introduction select the Main Wiki Page link to get to the user documentation:
```
Comment (judyj) [inline README.md]: Do we want this twice?
Once under "beginning with" and once under "setup?"

File: README.md [hunk 2]
```diff
@@ -81,75 +90,66 @@ Make sure that you have all Certificate Authorities needed loaded into SIMP.  If
 uses a different CA from yours, make sure you load their CA into your CA listing in PKI.  (See the SIMP
```
Comment (judyj) [inline README.md]: link here?

**Review by judyj — APPROVED** [review]

**Pull request merged**